sdk/core/Azure.Core/src/Shared/LightweightPkcs8Decoder.cs

```diff
@@ -20,7 +21,7 @@ namespace Azure.Core
     ///
     /// This code is able to decode RSA keys (without any attributes) from well formed PKCS#8 blobs.
     /// </summary>
-    internal class LightweightPkcs8Decoder
+    internal partial class LightweightPkcs8Decoder
```

Comment:
```suggestion
    internal partial static class LightweightPkcs8Decoder
```

Reply:
Good catch. I already made the change locally and will get it in mop up.